test/README.md

```diff
@@ -75,7 +75,6 @@ You can set these environment variables to configure the test suite:
                   "centos-8-stream"
                   "debian-stable"
                   "debian-testing"
-                  "fedora-31"
```

Comment:
we still have the image, but lets drop it from here already. Now just other projects like c-composer and c-podman test on it. For some, like certificates, it is the only supported OS so there should be some migration first.